### ISSUE TITLE
impl Shape for Arc

### ISSUE THREAD

**Issue by derekdreery** [linebender/kurbo#103]
I'm wanting to use `Arc`s for a UI and I notice that it doesn't implement `Shape`, so I did the following to get what I wanted:

```rust
fn arc(arc: Arc) -> BezPath {
    // this is hard-coded for my choice of x_rotation
    let start = Point {
        x: (1.0 - arc.start_angle.sin()) * arc.center.x,
        y: (2.0 + (arc.start_angle.cos() - 1.0)) * arc.center.y,
    };

    let mut out = BezPath::new();
    out.move_to(start);
    for el in arc.append_iter(1.0) {
        out.push(el);
    }
    out
}
```

Is there anything preventing an implementation of `Shape` for `Arc`?

**Comment by cmyr** [comment]
cc @raphlinus

**Comment by raphlinus** [comment]
Sure, I think it's possible. Of course, the methods (such as area and winding number) that expect a closed shape won't be meaningful.

**Comment by derekdreery** [comment]
This was fixed in #111.
